src/cdk/overlay/dispatchers/overlay-keyboard-dispatcher.ts

```diff
@@ -18,7 +18,7 @@ import {BaseOverlayDispatcher} from './base-overlay-dispatcher';
  */
 @Injectable({providedIn: 'root'})
 export class OverlayKeyboardDispatcher extends BaseOverlayDispatcher {
-  constructor(@Inject(DOCUMENT) document: any) {
+  constructor(@Inject(DOCUMENT) document: any, private _ngZone: NgZone) {
```

Comment:
Since these APIs are public, the `_ngZone` parameter will have to be optional, otherwise it's a breaking change for anybody extending the class. Here's an example of how we do it in other places: https://github.com/angular/components/blob/master/src/cdk/text-field/autosize.ts#L123

Reply:
Thanks, updated.